builder/ConfTargets.cmake

```diff
@@ -109,6 +109,7 @@ cmake_dependent_option(
 option( MFX_ENABLE_VP9_VIDEO_ENCODE "Enable VP9 encoder?" ON)
 
 option( MFX_ENABLE_MCTF "Build with MCTF support?"  ${MFX_1_25_OPTIONS_ALLOWED} )
+option( MFX_ENABLE_ASC "Build with ASC support?"  ON )
```

Comment:
MCTF should be transformed to dependent option from ASC and binary kernels, per @spossos it won't work without them:
```
cmake_dependent_option(
  MFX_ENABLE_MCTF "Enable MCTF?" ON
  "MFX_ENABLE_ASC;MFX_ENABLE_KERNELS" OFF)
```
And please, use word "Enable" rather than build: "Enable ASC?"

Reply:
I will do task 2) wrapping of features that depened on ASC in AVC